### PR TITLE
Conveniences found by AoC

### PR DESCRIPTION
**Pull request by stylewarning**
Small, miscellaneous conveniences will be collected here.

### CODE REVIEW

File: src/library/more-instances.lisp [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(in-package #:coalton-library)
+
+(coalton-toplevel
```
Comment (eliaslfox) [inline src/library/more-instances.lisp:3]: Is there a reason this can't be in classes.lisp?

Comment (stylewarning) [reply src/library/more-instances.lisp:3]: `not` wasn't defined earlier enough. Now it's a non-problem.

**Comment by stylewarning** [comment]
The commit history has been cleaned and made sort of nice.

This PR adds functionality, but also fixes lots of little tech debt things (like ~5 `STYLE-WARNING`s)

The build failure looks like a failure to download SBCL... tests pass locally for me...

**Review by eliaslfox — APPROVED** [review]
Looks good

**Pull request merged**